inference-engine/src/mkldnn_plugin/nodes/proposal_imp.cpp

```diff
@@ -13,6 +13,7 @@
 #if defined(HAVE_AVX2)
 #include <immintrin.h>
 #endif
+#include <ie_iextension.h>
```

Comment:
Looks like misprint. Should be `<ie_exception.h>`.

Reply:
Moreover, the concept of cross compiled sources was in avoiding of usage IE specific header in "_impl" cross compiled files. Because it may potentially lead to "illegal instruction" injection into final binary.
I suggest to use C like way of handling exceptional situation (e.g. return error code instead of void).

Reply:
Fixed